### PR TITLE
Feature/private repo

### PR DESCRIPTION
**Pull request by blankdots**
enable option for adding private registry

### CODE REVIEW

File: sda-db/templates/_helpers.tpl [hunk 1]
```diff
@@ -37,3 +37,7 @@ Create chart name and version as used by the chart label.
 {{- define "pgOutPassword" -}}
     {{- ternary (randAlphaNum 12) .Values.global.pg_out_password (empty .Values.global.pg_out_password) -}}
 {{- end -}}
+
+{{- define "image.registry" -}}
+{{- ternary "" (printf "%s/" .Values.image.registry) (empty .Values.image.registry) -}}
+{{- end -}}
```
Comment (vvaltia) [inline sda-db/templates/_helpers.tpl:43]: Nice way of doing this. No need to worry about `/`

File: sda-svc/values.yaml [hunk 2]
```diff
@@ -3,6 +3,9 @@
 # Declare variables to be passed into your templates.
 global:
 
+# Image registry if an private registry is used
+  imageRegistry: ""
```
Comment (vvaltia) [inline sda-svc/values.yaml:7]: Should this be `image.registry` ?

**Review by vvaltia — APPROVED** [review]

File: sda-svc/values.yaml (hunk 2 above)
Comment (blankdots) [reply sda-svc/values.yaml:7]: to be consistent sure.

Comment (vvaltia) [reply sda-svc/values.yaml]: I might be mistaken, but with a quick glance in the template files `image.registry` was used, and this might have to match it

**Pull request merged**